core/modules/catalog/helpers/tax.ts

```diff
@@ -1,287 +0,0 @@
-
-          if (deprecatedPriceFieldsSupport) {
-            /** BEGIN @deprecated - inconsitent naming kept just for the backward compatibility */
-            configurableChild.originalPrice = configurableChild.original_price
-            configurableChild.originalPriceInclTax = configurableChild.original_price_incl_tax
-            configurableChild.originalPriceTax = configurableChild.original_price_tax
-            configurableChild.priceInclTax = configurableChild.price_incl_tax
-            configurableChild.priceTax = configurableChild.price_tax
-            /** END */
-          }
-        }
-      } else {
-        configurableChild.special_price = 0
-      }
-
-      if ((configurableChild.price_incl_tax <= product.price_incl_tax) || product.price === 0) { // always show the lowest price
-        product.price_incl_tax = configurableChild.price_incl_tax
-        product.price_tax = configurableChild.price_tax
-        product.price = configurableChild.price
-        product.special_price = configurableChild.special_price
-        product.special_price_incl_tax = configurableChild.special_price_incl_tax
-        product.special_price_tax = configurableChild.special_price_tax
-        product.original_price = configurableChild.original_price
-        product.original_price_incl_tax = configurableChild.original_price_incl_tax
-        product.original_price_tax = configurableChild.original_price_tax
-
-        if (deprecatedPriceFieldsSupport) {
-          /** BEGIN @deprecated - inconsitent naming kept just for the backward compatibility */
-          product.priceInclTax = product.price_incl_tax
-          product.priceTax = product.price_tax
-          product.specialPriceInclTax = product.special_price_incl_tax
-          product.specialPriceTax = product.special_price_tax
-          product.originalPrice = product.original_price
-          product.originalPriceInclTax = product.original_price_incl_tax
-          product.originalPriceTax = product.original_price_tax
-          /** END */
-        }
-      }
-    }
-  }
-}
-
-export function isUserGroupedTaxActive (): boolean {
-  if (typeof currentStoreView().tax.userGroupId === 'number') {
-    return true
-  }
-  return true
-}
-
-export function getUserTaxGroupId (): number|void {
-  if (!isUserGroupedTaxActive()) return
-
-  if (currentStoreView().tax.useOnlyDefaultUserGroupId) return currentStoreView().tax.userGroupId
-
-  return rootStore.getters['user/isLoggedIn'] === true ? rootStore.state.user.current.group_id : currentStoreView().tax.userGroupId
-}
-
-export function calculateProductTax (product, taxClasses, taxCountry = 'PL', taxRegion = '', sourcePriceInclTax = false, deprecatedPriceFieldsSupport = false, finalPriceInclTax = true) {
-  let rateFound = false
-  if (product.tax_class_id > 0) {
-    let taxClass
-    if (typeof currentStoreView().tax.userGroupId === 'number') {
-      const userGroupId = getUserTaxGroupId()
-
-      taxClass = taxClasses.find((el) => {
```

Comment:
soo basically this file was replaced by mirror copy of that https://github.com/DivanteLtd/vue-storefront-api/blob/master/src/lib/taxcalc.js because as far as I know it should be up to date with this one

Reply:
When you look into the repo and develop branch there I made the exact same change as I did here to get the taxes right for magento stores that have more complex tax rules which need the customer group id to work.  

https://github.com/DivanteLtd/vue-storefront-api/blob/develop/src/lib/taxcalc.js#L213

So it would be nice when you could bring that back into the code and not remove that feature.

Reply:
yeah soo I used a wrong branch... ok sure